cypress/e2e/homepage.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { ecosystemLinks, resourcesLinks } from "../fixture/homepage.config";
+
+const viewports = ["iphone-xr", "macbook-16"];
```

Comment:
we have UI for tablet, should we include tablet viewport too?

Reply:
to be expected in the separate PR